### DISCUSSION TITLE
Berkeley QA Release a4eae89

### DISCUSSION THREAD

**Discussion by deepthiskumar** [MinaProtocol/mina#10890] [Berkeley]
Mina's ecosystem partner, O(1) Labs, is continuously working on improvements to the core protocol and today we are excited to announce an updated unstable release `a4eae89` that is ready to test on the Berkeley QANet! Please report any issues in the comments on this post or tag any issues you create with `berkeley-qa` and include berkeley `a4eae89` in the issue title.

### New Features/Improvements:
- Tokens feature to enable creating accounts and transacting in non-default tokens. This changes the parties transaction structure and the graphQL schema.
Note: zkApps Test Transaction CLI tool cannot be used to create token accounts in this release.

### Fixes:
- Archive processor bugs in QA net
https://github.com/MinaProtocol/mina/issues/10343
- Invalid fee excess bug that halted the QA-net
https://github.com/MinaProtocol/mina/issues/10843

### Complete Changelog since `69730d8` in:
Github Compare: https://github.com/MinaProtocol/mina/compare/69730d8f7db97861e028e4bf73fa255a2dd45a13...a4eae892e1019f82708d34b12164b9f51465189e

### Debian Packages:
**Debian Repository:**
First, set up and update the `unstable` Debian Repository for your platform. Replace the word `CODENAME` with the appropriate codename for your machine, one of `bionic`, `focal`, `stretch`, `buster`, or `bullseye` and run:
```
echo "deb [trusted=yes] http://packages.o1test.net CODENAME unstable" | sudo tee /etc/apt/sources.list.d/mina-unstable.list
sudo apt-get update
```
Then, install the package(s) that you need:
**Mina Daemon:** `sudo apt-get install -y mina-berkeley=1.3.1beta1-release-2.0.0-a4eae89`
**zkApps Test Transaction CLI:** `sudo apt-get install -y mina-zkapp-test-transaction=1.3.1beta1-release-2.0.0-a4eae89`
[ zkApps Transaction CLI Documentation](https://github.com/MinaProtocol/mina/blob/ecbb3d490dce0f199894ebdf787a03b8fec64bd4/src/app/zkapp_test_transaction/README.md)

### Docker Images:
Every docker image is built for each of the supported platforms, replace `CODENAME` in the images below with your choice of `bionic`, `focal`, `stretch`, `buster`, or `bullseye` to select the base docker image.
**Mina Daemon:** `minaprotocol/mina-daemon:1.3.1beta1-release-2.0.0-a4eae89-CODENAME-berkeley`

### Step by Step Guide:
This version will only connect and function properly on the newly launched Berkeley QA Network. *Docker only* Make sure to run with `--peer-list-url https://storage.googleapis.com/seed-lists/berkeley_seeds.txt`.

If you are running the correct version on the correct network **BERKELEY QA NET**, `mina client status` will show:
```
Chain id:     44160dad015e5faabd954a053060cfe7d8ff87e8244640b0433c9374d37b9609
Git SHA-1:    a4eae892e1019f82708d34b12164b9f51465189e
```

**Comment by duccas** [comment]
I think versions on this post is wrong.
Need to use:
```
sudo apt-get install -y mina-berkeley=1.3.1beta1-release-2.0.0-a4eae89
sudo apt-get install -y mina-zkapp-test-transaction=1.3.1beta1-release-2.0.0-a4eae89
minaprotocol/mina-daemon:1.3.1beta1-release-2.0.0-a4eae89-CODENAME-berkeley
```

**Comment by p-shahi** [comment]
@icohigh Yes that's right. I'll update the notes
